# goal
geometry_msgs/Point start     # arm start position
geometry_msgs/Point normal    # vector normal to direction of approach
float32 search_distance       # How deep along normal vector the scoop moves
---
# result
geometry_msgs/Point final     # arm final position
bool success
---
# feedback
geometry_msgs/Point current   # arm current position
